orchestrator/user/session.go

```diff
@@ -62,6 +62,20 @@ func (m *SessionManager) Get(request *http.Request) *SessionData {
 	return &session.Data
 }
 
+func (m *SessionManager) Destroy(response http.ResponseWriter, request *http.Request) {
+	sessionID := getSessionCookie(request)
+	if sessionID != "" {
+		m.store.destroy(sessionID)
+	}
```

Comment:
We should probably do something when the session is not found, either log a warning or maybe even return an error?

Reply:
For now let's log a warning, we can use those logs to determine how often it is happening and if we need to error out.

For now I don't see any harm in logout being called without a session open but maybe I am missing something?